src/main/scala/com/gu/scanamo/package.scala

```diff
@@ -4,7 +4,7 @@ import com.gu.scanamo.query._
 
 package object scanamo {
 
-  object syntax {
+  object syntax extends Scannable.ToScannableOps with Queryable.ToQueryableOps {
```

Comment:
Just importing an op from another object doesn't export into the implicit scope when importing `object._`